src/bin/dtntrigger.rs

```diff
@@ -28,10 +29,22 @@ impl Connection {
     }
     fn execute_cmd(&self, data_file: NamedTempFile, bndl: &Bundle) -> Result<()> {
         let fname_param = format!("{}", data_file.path().display());
-        let output = Command::new(&self.command)
+        let cmd_args = &mut self.command.split(" ");
+        let mut command = Command::new(cmd_args.next().unwrap());
+        while let Some(arg) = cmd_args.next() {
+            command.arg(arg);
+        }
+        let output = match command
             .arg(bndl.primary.source.to_string())
             .arg(fname_param)
-            .output()?;
+            .output() 
+            {
```

Comment:
Turn this into an 
```suggestion
          .unwrap_or_else(
                |e| {
                    eprintln!("Error executing command: {}", e);
                    std::process::exit(1);
                }); 
```
and remove the `match` which is not necessary anymore